packages/react-native-reanimated/src/workletFunctions.ts

```diff
@@ -0,0 +1,70 @@
+import type {
+  MakeShareableClone,
+  WorkletRuntime as WorkletRuntimeFromWorklets,
+} from 'react-native-worklets';
+import {
+  createWorkletRuntime as createWorkletRuntimeFromWorklets,
+  executeOnUIRuntimeSync as executeOnUIRuntimeSyncFromWorklets,
+  isWorkletFunction as isWorkletFunctionFromWorklets,
+  makeShareableCloneRecursive as makeShareableCloneRecursiveFromWorklets,
+  runOnJS as runOnJSFromWorklets,
+  runOnRuntime as runOnRuntimeFromWorklets,
+  runOnUI as runOnUIFromWorklets,
+} from 'react-native-worklets';
+
+/**
+ * @deprecated This function is deprecated and will be removed in the next major
+ *   Please import `makeShareableCloneRecursive` directly from
```

Comment:
This would be enough

```suggestion
/**
 * @deprecated Please import `makeShareableCloneRecursive` directly from
 *   `react-native-worklets` instead of `react-native-reanimated`.
 */
```